AEPServices/Sources/ui/floating/FloatingButtonPosition.swift

```diff
@@ -0,0 +1,44 @@
+/*
+ Copyright 2021 Adobe. All rights reserved.
+ This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License. You may obtain a copy
+ of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software distributed under
+ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ OF ANY KIND, either express or implied. See the License for the specific language
+ governing permissions and limitations under the License.
+ */
+
+import Foundation
+import UIKit
+
+@objc(AEPFloatingButtonPosition) public enum FloatingButtonPosition: Int {
```

Comment:
This will be exposed to customers also do we want to do that?

Reply:
Yes this was intended, for anyone who creates a floatingButton to change its initial show position